### ISSUE TITLE
No plot for channel proscan-archive/BMA1:STA:2

### ISSUE THREAD

**Issue by djlauk** [paulscherrerinstitute/databuffer-ui#47]
### What I did
* Navigate to https://data-ui-beta.psi.ch/search
* Search for "BMA1:STA:2"
* Select channel "proscan-archive/BMA1:STA:2"
* Click button "Plot selected"
* Select time "last 1h"
* Click button "Plot"

### What happened
* Plot appears, but is empty
* There's an error message in the browser console:

```
Uncaught (in promise) Error: Highcharts error #14: www.highcharts.com/errors/14/
    at t.Chart.l (highcharts.js:12)
    at Object.t.fireEvent (highcharts.js:32)
    at t.error (highcharts.js:12)
    at i.setData (highcharts.js:356)
    at i.update (highcharts.js:414)
    at highcharts.js:406
    at Array.forEach (<anonymous>)
    at highcharts.js:406
    at Array.forEach (<anonymous>)
    at t.Chart.update (highcharts.js:406)
l @ databuffer-ui.min.js:1
t.fireEvent @ databuffer-ui.min.js:1
t.error @ databuffer-ui.min.js:1
setData @ databuffer-ui.min.js:1
update @ databuffer-ui.min.js:1
(anonymous) @ databuffer-ui.min.js:1
(anonymous) @ databuffer-ui.min.js:1
update @ databuffer-ui.min.js:1
updated @ view-channel-search.ts:31
performUpdate @ lit-html.js:59
_enqueueUpdate @ lit-html.js:59
async function (async)
_enqueueUpdate @ lit-html.js:59
_requestUpdate @ lit-html.js:59
set @ lit-html.js:59
(anonymous) @ unsafe-html.js:21
u @ databuffer-ui.min.js:1
e @ VM58:1
(anonymous) @ module.js:5
(anonymous) @ module.js:5
(anonymous) @ databuffer-ui.min.js:1
dispatch @ databuffer-ui.min.js:1
(anonymous) @ databuffer-ui.min.js:1
(anonymous) @ databuffer-ui.min.js:1
de @ databuffer-ui.min.js:1
fe @ databuffer-ui.min.js:1
ce @ databuffer-ui.min.js:1
Ae.PUT @ databuffer-ui.min.js:1
(anonymous) @ databuffer-ui.min.js:1
u @ databuffer-ui.min.js:1
h @ databuffer-ui.min.js:1
l @ databuffer-ui.min.js:1
Promise.then (async)
Se @ databuffer-ui.min.js:1
Ae.CALL @ databuffer-ui.min.js:1
(anonymous) @ databuffer-ui.min.js:1
u @ databuffer-ui.min.js:1
h @ databuffer-ui.min.js:1
l @ databuffer-ui.min.js:1
(anonymous) @ databuffer-ui.min.js:1
de @ databuffer-ui.min.js:1
fe @ databuffer-ui.min.js:1
ce @ databuffer-ui.min.js:1
Ee.a.put @ databuffer-ui.min.js:1
(anonymous) @ module.js:5
(anonymous) @ databuffer-ui.min.js:1
dispatch @ VM58:1
<computed>.<computed> @ unsafe-html.js:21
__plot @ view-channel-search.ts:31
handleEvent @ template-result.js:110
In.__boundHandleEvent @ template-result.js:110
```

### What I expected to happen

* Plot with values appears
* No errors in console

**Comment by djlauk** [comment]
**Note to self:** This is an enum, which the query backend reports back as a `string` value.

Pointers on plotting string values with Highcharts:

- https://api.highcharts.com/highcharts/yAxis.type
- https://api.highcharts.com/highcharts/xAxis.categories
- http://jsfiddle.net/x22q0f8m/

**Comment by djlauk** [comment]
Has been solved together with issue #45
